src/Microsoft.Identity.Client/Internal/Cache/BaseTokenCacheItem.cs

```diff
@@ -36,78 +36,32 @@ namespace Microsoft.Identity.Client.Internal.Cache
     [DataContract]
     internal abstract class BaseTokenCacheItem
     {
-        protected RequestContext RequestContext;
-
         /// <summary>
         /// Default constructor.
         /// </summary>
-        internal BaseTokenCacheItem(string authority, string clientId, TokenResponse response)
+        public BaseTokenCacheItem(string clientId)
```

Comment:
for serialization/deserialization

---
In reply to: [109344344](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/282#discussion_r109344344) [](ancestors = 109344344)